code/datums/shuttles.dm

```diff
@@ -554,6 +558,14 @@
 	suffix = "default"
 	name = "pirate ship (Default)"
 
+/datum/map_template/shuttle/hunter/space_cop
+	suffix = "space_cop"
+	name = "Police Spacevan"
+
+/datum/map_template/shuttle/hunter/russian
```

Comment:
there's still something wrong with these that makes the map template not load. dnm until i work that out

Reply:
So is this in working state now or still needs something ?

Reply:
got it, finally...